specification/migrate/resource-manager/Microsoft.Migrate/stable/2018-02-02/examples/LocationCheckNameAvailability_AlreadyExists.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "api-version": "2018-02-02",
```

Comment:
Why we change `2018-02-02` API version in this PR?

Reply:
@sergey-shandar, we are targeting Network-September-Release while the branch is based on the latest master. So we have extra commit there.
@MikhailTryakhov, please sync Network-September-Release branch with master to avoid confusion

Reply:
can't push changes to monthly branch directly.
@sergey-shandar  can you please help to approve https://github.com/Azure/azure-rest-api-specs/pull/3687 and resolve requested changes here?

Reply:
@EvgenyAgafonchikov @MikhailTryakhov I've merge the PR #3687. This PR still has a lot of irrelevant changes.